src/test/java/BaseTest.java

```diff
@@ -4,15 +4,66 @@
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
 import org.openqa.selenium.safari.SafariDriver;
+import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
 
+import java.time.Duration;
+
 
 public class BaseTest {
+    public static WebDriver driver = null;
 
     @BeforeSuite
-    static void setupClass() {
+    static void setupClass(){
         WebDriverManager.chromedriver().setup();
     }
+    @BeforeMethod
+    public static void setUpBrowser(){
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+    }
+    @AfterMethod
+    public static void closeBrowser(){
+        driver.quit();
+    }
+    public static void urlAccess() throws InterruptedException {
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+        driver.manage().window().maximize();
+        Thread.sleep(2000);
+    }
+
+    public static void getEmail(String email) throws InterruptedException {
+        WebElement insertEmail = driver.findElement(By.cssSelector("[type='email']"));
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/223046463-68c9a141-d929-4d11-a164-9a4cfdf7dc00.png)

**Hint** 

`import org.openqa.selenium.By;`